src/native/libs/System.Native.Browser/libSystem.Native.Browser.footer.js

```diff
@@ -37,7 +37,10 @@
         for (const exportName of Reflect.ownKeys(exports)) {
             const name = String(exportName);
             if (name === "dotnetInitializeModule") continue;
-            lib[name] = exports[name];
+            const fn = lib[name] = exports[name];
+            if(fn.__deps) {
+                lib[name + "__deps"] = fn.__deps;
+            }
```

Comment:
Why is this needed now?

Reply:
So that the C symbol I call from inside of the JS method is not trimmed by the emcc linker.
Note that emcc doesn't really parse the JS code.

in `src\native\libs\System.Native.Browser\native\timer.ts` I use the function as property bag.
I put `SystemJS_ScheduleTimer["__deps"] = ['SystemJS_ExecuteTimerCallback'];`
And here I'm lifting it into "library" level, which is what the `autoAddDeps` expects.

I still need to test that this works even with JS name mangling in Release configuration.

see also https://emscripten.org/docs/porting/connecting_cpp_and_javascript/Interacting-with-code.html#javascript-limits-in-library-files